core/src/components/datetime/datetime.tsx

```diff
@@ -322,6 +324,17 @@ export class Datetime implements ComponentInterface {
    */
   @Prop() multiple = false;
 
+  /**
+   * Used to apply custom text and background colors to specific dates.
+   *
+   * Can be either an array of objects containing ISO strings and colors,
+   * or a callback that receives an ISO string and returns the colors.
+   *
+   * Only applies to the `date`, `date-time`, and `time-date` presentations,
+   * with `preferWheel="false"`.
+   */
+  @Prop() highlightedDates?: DatetimeHighlight[] | DatetimeHighlightCallback;
```

Comment:
Our API conventions guide states that APIs should be present tense. Could we change this to `highlightDates` or something similar?

Reply:
`highlighted` is used as an adjective here, so it's still present tense, plus `highlightDates` sounds like it's the function that actually does the highlighting, like `highlight` as a verb. 🤔 If I'm still missing the point, maybe `datesWithHighlight`...? Or just `highlights`? I also don't mind `notableDates` from the bikeshedding Slack thread, although that might necessitate renaming the types too. (cc @sean-perkins in case you had any thoughts)

Reply:
Ah right "present tense" isn't the correct term here. However, our API conventions state that we should avoid this usage. For example, we try to do things like `onPresent` instead of `onPresented` to make things easier to reason about for non-native English speakers. This also aligns with how the browser APIs are named (for example `change` instead of `changed` for `<input />`)

Reply:
Oh I know, I'll go with `dateHighlights` 👀

Reply:
Though thinking about this more... `onPresent` and `change` are events whereas this is a property name. Also we have `isDateEnabled` so maybe I'm overthinking this 😂 . `highlightedDates` is probably fine.